ai.py

```diff
@@ -36,4 +35,11 @@ def next(self, messages: list[dict[str, str]], prompt=None):
             msg = delta.get('content', '')
             print(msg, end="")
             chat.append(msg)
-        return messages + [{"role": "assistant", "content": "".join(chat)}]
\ No newline at end of file
+        return messages + [{"role": "assistant", "content": "".join(chat)}]  # Add assistant's message to chat history
+
+# Create an instance of the AI class
+ai = AI()
+
+# Example usage:
+# Start a conversation with a system message and a user message
+ai.start("Hello, how can I assist you?", "Can you provide information about your products?")
```

Comment:
What's the point of these lines?

Reply:
I believe @AayushTech meant those as an example?

They don't belong here though, maybe in the `README.md` but I would get rid of them all together.
Also, the comments throughout the code are not that helpful, I believe comments should only exist in particular cases where the code for some reason is difficult to understand.

In this case I believe the code is simple enough